framework/database/seeds/CVsTableSeeder.php

```diff
@@ -10,7 +10,7 @@ class CVsTableSeeder extends Seeder
      */
     public function run()
     {
-        $users = \App\User::retrieveUsers();
+        $users = \App\User::all();
         foreach($users as $user)
         {
             factory(App\CV::class)->create(['user_id' => $user->id]);
```

Comment:
Quote:

Les tables de liaison triple, ce n'est pas ce que Eloquent (l'ORM) sait faire de mieux. Néanmoins, ceci devrait le faire:

```php
$skill = factory(App\Skill::class)->create();
$user->skills()->save($skill);
$section->skills()->save($skill);
```

----
En suivant la logique:
1. on cree un modele skill et on l'envoit à la bdd.
2. on prend tous les skills du user et on ajoute le skill crée selon les cléfs. Lesqueles? Pas specifé, donc, whatever.
3. Pour chaque section, on cherche les skills et on ajoute le skill crée.

----

Même logique suivie pour le code montré avant. 

----
Save: Save the model to the database.
Create: Create a collection of models and persist them to the database.
Make: Create a collection of models.

Donc, le plus logique, on crée le modèle, et on l'ajoute après? Ou je ne comprends vraiment rien?